server/@types/viewModels/index.d.ts

```diff
@@ -112,6 +112,7 @@ declare module 'viewModels' {
     courseStartDate: Date
     courseStatus: 'COMPLETED' | 'IN_PROGRESS' | 'WITHDRAWN' | 'TEMPORARILY_WITHDRAWN'
     courseCompletionDate?: Date
+    coursePlannedEndDate?: Date
```

Comment:
I had to add `coursePlannedEndDate` to the view. model class, as it wasnt there before; hence the changes below to the mapper and it's tests